src/hotspot/share/oops/methodFlags.hpp

```diff
@@ -58,6 +58,7 @@ class MethodFlags {
    status(has_loops_flag              , 1 << 13) /* Method has loops */ \
    status(has_loops_flag_init         , 1 << 14) /* The loop flag has been initialized */ \
    status(on_stack_flag               , 1 << 15) /* RedefineClasses support to keep Metadata from being cleaned */ \
+   status(has_matching_directives     , 1 << 16) /* The method has matching directives */ \
```

Comment:
It's worth noting that the flag is temporary and is valid only during DCmd execution.

Reply:
Good point, updated the comment. This btw means that in another places this flag can be reused.